pages/index.js

```diff
@@ -1,3 +1,21 @@
+import useSWR from "swr";
+import EventCard from "../components/EventCard";
+import { fetcher } from "../helpers/api";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data: events, error } = useSWR("/api/events", fetcher);
+  if (error) return <h1>There was an error</h1>;
+  if (!events) return <h1>...Loading...</h1>;
```

Comment:
I suggest to add an empty line for better readability 🙂 
```suggestion
  const { data: events, error } = useSWR("/api/events", fetcher);
  
  if (error) return <h1>There was an error</h1>;
  if (!events) return <h1>...Loading...</h1>;
```